Makefile

```diff
@@ -1,11 +1,26 @@
-all: generate test lint
+SHELL := /usr/bin/env bash
+
+GOGC ?= 1000 # Reduce GC frequency during testing, default to 1000 if unset.
+
+all: generate test fuzz lint
 
-test: GOGC ?= 1000 # Reduce the GC frequency, default to 1000 if unset.
 test:
 	GOGC=$(GOGC) go test ./...
 .PHONY: test
 
+fuzz: FUZZTIME ?= 10s # The duration to run fuzz testing, default to 10s if unset.
+fuzz: # List all fuzz tests across the repo, and run them one at a time with the configured fuzztime.
+	@go run ./scripts/list_fuzz_tests | while read -r line; do \
```

Comment:
For future readers, this _is_ parallelized because go will internally parallelize fuzzing.